indexer/index_helpers.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "std/function.hpp"
+
+class Index;
+class FeatureType;
+
+namespace indexer
+{
+using TFeatureTypeFn = function<void(FeatureType &)>;
```

Comment:
Not good. The name is very generic, it's dangerous to put such things directly into indexer namespace. Please, use function<void ...> directly, without typedef.